### PR TITLE
Fixed filter resetting.

### PR DESCRIPTION
**Pull request by jorenham**

### CODE REVIEW

**Comment by alangerak** [comment]
looks good

File: lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java [hunk 1]
```diff
@@ -122,6 +122,7 @@ private void registerShoutListeners() {
 
         listen(Message.RESET, (sender, subject, args) -> {
             updateVisible(new HashSet<Sequence>(sequences.values()));
+            shout(Message.FILTERED, "", new HashSet<>(sequences.values()));
```
Comment (rutgervandenberg) [inline lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java]: should save the first one

**Comment by rutgervandenberg** [comment]
Formatter ;)

File: lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java [hunk 2]
```diff
@@ -121,7 +121,9 @@ private void registerShoutListeners() {
         });
 
         listen(Message.RESET, (sender, subject, args) -> {
-            updateVisible(new HashSet<Sequence>(sequences.values()));
+            HashSet<Sequence> visibleSet = new HashSet<>(sequences.values());
```
Comment (rutgervandenberg) [inline lifetiles-sequence/src/main/java/nl/tudelft/lifetiles/sequence/controller/SequenceController.java]: Program for interface not implementation

**Comment by rutgervandenberg** [comment]
Looks good now

**Pull request merged**